Formula/v/vte3.rb

```diff
@@ -33,17 +32,27 @@ class Vte3 < Formula
   depends_on "pcre2"
 
   on_macos do
-    depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1200
     depends_on "gettext"
   end
 
+  on_ventura :or_newer do
+    depends_on "llvm" => :build if DevelopmentTools.clang_build_version <= 1500
+  end
+
+  on_monterey :or_older do
+    # We use GCC on older macOS as build fails with brew `llvm`.
+    # Undefined symbols for architecture x86_64:
+    #   "std::__1::__libcpp_verbose_abort(char const*, ...)", referenced from: ...
+    depends_on "gcc"
```

Comment:
maybe adding `fails_with :clang` in here?

```suggestion
    depends_on "gcc"
    fails_with :clang
```

Reply:
We already have
```rb
  fails_with :clang do
    build 1500
```

which means fails `<= 1500` so already covered.